Tools/autotest/common.py

```diff
@@ -1408,14 +1408,14 @@ def initialise_after_reboot_sitl(self):
         self.set_streamrate(self.sitl_streamrate())
         self.progress("Reboot complete")
 
-    def customise_SITL_commandline(self, customisations, model=None, defaults_filepath=None):
+    def customise_SITL_commandline(self, customisations, model=None, defaults_filepath=None, wipe=False):
         '''customisations could be "--uartF=sim:nmea" '''
         self.contexts[-1].sitl_commandline_customised = True
         self.stop_SITL()
         self.start_SITL(model=model,
                         defaults_filepath=defaults_filepath,
                         customisations=customisations,
-                        wipe=False)
+                        wipe=wipe)
```

Comment:
Does "wipe=wipe" actually work?

Reply:
It does, because the first one isn't a variable name; it's a Pythonesque named-argument.

I do agree it looks a bit odd.